src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Declarations_SourceDeclarations.cs

```diff
@@ -56,5 +52,50 @@ public static Task<IEnumerable<ISymbol>> FindSourceDeclarationsAsync(Project pro
                 return declarations.SelectAsArray(t => t.Symbol);
             }
         }
+
+#pragma warning disable RS0026
+
+        /// <summary>
+        /// Find the symbols for declarations made in source with the specified name.
+        /// </summary>
+        public static Task<IEnumerable<ISymbol>> FindSourceDeclarationsWithPatternAsync(Solution solution, string pattern, CancellationToken cancellationToken = default)
```

Comment:
:bulb: The API would be improved if `cancellationToken` was not optional

Reply:
That would make it inconsistent with the rest of our public API surface area.